Libplanet/Net/Swarm.cs

```diff
@@ -1263,6 +1435,58 @@ private void TransferBlocks(GetBlocks getData)
             _logger.Debug("Transfer complete.");
         }
 
+        private void TransferRecentStates(GetRecentStates getRecentStates)
+        {
+            HashDigest<SHA256> blockHash = getRecentStates.BlockHash;
+            IImmutableDictionary<HashDigest<SHA256>,
+                IImmutableDictionary<Address, object>
+            > blockStates = null;
+            IImmutableDictionary<Address, IImmutableList<HashDigest<SHA256>>>
+                stateRefs = null;
+            IImmutableDictionary<Address, long> txNonces = null;
+
+            if (_blockChain.Blocks.ContainsKey(blockHash))
+            {
+                // FIXME: Swarm should not directly access to the IStore instance,
+                // but BlockChain<T> should have an indirect interface to its underlying
+                // store.
+                IStore store = _blockChain.Store;
+                string ns = _blockChain.Id.ToString();
+
+                IImmutableSet<Address> addresses =
```

Comment:
Do we need this variable? It is only used when creating stateRefs and seems to be no longer used. (copied from https://github.com/planetarium/libplanet/pull/343#discussion_r302828043)

Reply:
Indeed this variable and its realization into the concrete set is no more necessary.  Fixed: 4a4e5617d396a93cbe5b002167c50ac7824290f5